examples/next-prisma-todomvc/src/pages/[filter].tsx

```diff
@@ -305,7 +305,7 @@ export default function TodosPage({
             <button
               className="clear-completed"
               onClick={() => {
-                clearCompleted.mutate(null);
+                clearCompleted.mutate();
```

Comment:
You need to do this change in a follow-up as the example deploy requires a new version to be published.

At the same time, having this here makes sure that you're not making any breaking changes

```suggestion
                clearCompleted.mutate(null);
```